### ISSUE TITLE
GPS location does not show up on map

### ISSUE THREAD

**Issue by hiwad-aziz** [ioBroker/ioBroker.lovelace#125]
Hi everyone!

I tried to add a GPS location to Lovelace but it does not show up on the map.

Lovelace version: 1.3.3
js-controller version: 3.1.6

These were my steps:
Created a state according to the README:
```
createState('Hiwad', HiwadGpsLoc, false, {
    "name": "location",
    "role": "value.gps",
    "type": "string",
    "read": true,
    "write": false,
    "desc": "Gps Coordinates",
    "custom": {
        "lovelace.0": {
            "enabled": true,
            "entity": "HiwadLoc",
            "name": "GPS"
        }
    }
});
```

The variable HiwadGpsLoc is a data point that's being updated on a regular basis. Value inside is in following form: 31.2350532;6.6764545

Configuration of my map:

```
type: map
entities:
  - entity: HiwadLoc.GPS
hours_to_show: 1
dark_mode: false
geo_location_sources: []
```

Anything that I am doing wrong?

**Comment by Garfonso** [comment]
TBH I didn't know this part of the readme.. ;-)
GPS Location on the map works with the auto detection and ioBroker device type `location`. This currently seems to work only for separate latitude and logitude states (although rough support for a combined state exists). 

Also the entity_id will then be very different, more like geo_location.NAMEOFDEVICE.

**Comment by hiwad-aziz** [comment]
Hi @Garfonso,
thanks for your response!
I followed your suggestion and created a device of type `location` which is also being detected automatically as geo_location.XX.

> This currently seems to work only for separate latitude and logitude states (although rough support for a combined state exists).

I am only able to create a device that takes in the GPS location as a value.gps type (so both lat and long combined). Tried to use the state that I've created myself (s. first comment) but it still does not show up in the map.
My device manager version is: 0.3.13.
Is there any other way to create a device of type `location` which can be configured with separate lat and lon?

**Comment by thost96** [comment]
Hi @hiwad-aziz,

as version 1.3.6 got released today with the note "Added: Support location devices with one GPS state in string form", could you test this again? I'm also having a GPS Device with the values as string and it seems to work for me.

**Comment by Garfonso** [comment]
@thost96 thank you for the feedback 😃

**Comment by hiwad-aziz** [comment]
Works like a charm now. Thanks @thost96 and @Garfonso!

**Comment by TobiJust** [comment]
Hi @hiwad-aziz 
Can you give more insights on the object you created with the type location?
I can't get it to work on my instance. 
The example in the README.md doesn't seem to work..

**Comment by TobiJust** [comment]
> Hi @hiwad-aziz
> Can you give more insights on the object you created with the type location?
> I can't get it to work on my instance.
> The example in the README.md doesn't seem to work..

Nevermind, I managed to get it work. 
I created a new device with the device in the device management https://github.com/ioBroker/ioBroker.devices with type location. Initially I set a gps string as value, but this was not taken into account and I needed to put the value manually inside the objects view. After that, the map in Lovelace displays the location.
